### PR TITLE
Add delete button and skeleton code for backend

### PR DESCRIPTION
**Pull request by fjuren**

### CODE REVIEW

File: server/routes/todo.js [hunk 1]
```diff
@@ -10,6 +10,9 @@ router.post("/todo", todoController.todo_post);
 router.get("/todo", todoController.todo_get);
 
 // Create a route to mark item as complete (we could also use this if we wanted to mark the item as not complete)
-router.put("/todo/:id", todoController.todo_complete);
+router.put("/todo/update/:id", todoController.todo_complete);
```
Comment (matthew-defusco) [inline server/routes/todo.js:13]: I think the previous route was correct.

With REST APIs, the method verb (which in this case would be a "PUT") is descriptive enough of the action you're taking so you shouldn't need to add anything different in the path. The previous path ("/todo/:id") represents the resource on the backend you're trying to access (something in a todos collection with a specific ID).

File: server/routes/todo.js [hunk 2]
```diff
@@ -10,6 +10,9 @@ router.post("/todo", todoController.todo_post);
...
+
+// Delete todo item
+router.post("/todo/delete/:id", todoController.todo_delete);
```
Comment (matthew-defusco) [inline server/routes/todo.js:16]: This should be a router.delete() call instead. And similar to my other comment, the path can just be a description of the resource you're looking for so it should be "/todo/:id".

File: server/routes/todo.js (hunk 1 above)
Comment (fjuren) [reply server/routes/todo.js:13]: Ok I'll make that change! I'm not on the computer at the moment but I'll update today

File: server/routes/todo.js (hunk 2 above)
Comment (fjuren) [reply server/routes/todo.js:16]: I was also wondering about this. Thanks man. Will update

**Comment by fjuren** [comment]
FYI the name of this PR is a bit misleading. I also did work on the backend. In hindsight I probably should have created another branch from the initial branch waiting to be reviewed...

**Review by matthew-defusco — APPROVED** [review]

**Pull request merged**